cmd/gossamer/configcmd.go

```diff
@@ -84,7 +84,7 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 		return nil, nil, err
 	}
 
-	log.Info("🕸\t Configuring node...", "datadir", fig.Global.DataDir, "protocolID", string(gendata.ProtocolId), "bootnodes", fig.P2p.BootstrapNodes)
+	log.Info("🕸\t Configuring node...", "datadir", fig.Global.DataDir, "protocol", fig.P2p.ProtocolId, "bootnodes", fig.P2p.BootstrapNodes)
```

Comment:
should be protocolID:, raised https://github.com/ChainSafe/gossamer/issues/559 to have lint reporting things like this next time.

Reply:
It looks like you update this in #555 already... should it be addressed there?

Reply:
It looks like you update this in #555 already... should it be addressed there?

Reply:
I will resolve that in the ticket mentioned after this is merged.